.github/release.yml

```diff
@@ -0,0 +1,18 @@
+changelog:
+  exclude:
+    authors:
+      - engine-flutter-autoroll
+      - fluttergithubbot
+    labels:
+      - passed first triage
```

Comment:
This will exclude PRs that have the labels `passed first triage` / `passed secondary triage`?

Reply:
I don't think any PRs would have these labels, only issues.

Reply:
@devoncarew  @jmagman correct, this is still experimentation.